tests/regression/36-apron/98-array_octagon_keep_last.c

```diff
@@ -95,7 +97,9 @@ void example2(void) {
   // Values that may be read are 1 or 0
   assert(a[z] == 1); //UNKNOWN
   assert(a[z] == 0); //UNKNOWN
-  assert(a[z] == 2); //FAIL
+
+  // Relies on option sem.int.signed_overflow assume_none
+  assert(a[z] != 2); //UNKNOWN
```

Comment:
Would this not be `FAIL` then, given that we now have ` --set sem.int.signed_overflow assume_none` ?